gensim/models/ldaseqmodel.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, corpus=None, time_slice=None, id2word=None, alphas=0.01, num_
         time_slice : list of int, optional
             Number of documents in each time-slice. Each time slice could for example represent a year's published
             papers, in case the corpus comes from a journal publishing over multiple years.
-            It is asummed that `sum(time_slice) == num_topics`.
+            It is asummed that `sum(time_slice) == num_documents`.
```

Comment:
```suggestion
            It is assumed that `sum(time_slice) == num_documents`.
```

Reply:
Could you please fix this typo? Looks like I don't have permissions to push to your repo.